testpmd/hooks/pre-run.yml

```
@@ -1,8 +1,4 @@
 ---
-- name: "Set working namespace"
-  set_fact:
-    cnf_namespace: "example-cnf"
-
```

We are going to need this one as is used by the upstream repo ([nfv-example-cnf-deploy](https://github.com/rh-nfv-int/nfv-example-cnf-deploy)).
```suggestion
- name: "Set working namespace"
  set_fact:
    cnf_namespace: "example-cnf"

```

---

sure, I'll set the fact back